src/NHibernate/Linq/Visitors/ResultOperatorProcessors/ProcessFetch.cs

```diff
@@ -29,7 +29,8 @@ public void Process(FetchRequestBase resultOperator, QueryModelVisitor queryMode
 
 		private void Process(FetchRequestBase resultOperator, QueryModelVisitor queryModelVisitor, IntermediateHqlTree tree, HqlDot memberPath, IType propType)
 		{
-			if (resultOperator is FetchOneRequest)
+			string alias = null;
+			if (resultOperator is FetchOneRequest fetchRequest)
```

Comment:
`fetchRequest` variable seems not really required. `resultOperator` can still be used for comparison.

Reply:
Variable removed.